scripts/run-ui-tests

```diff
@@ -5,6 +5,15 @@ set -e -o pipefail -u
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
```

Comment:
3 (locally: `gradle test`, `gradle runIdeForUiTests` -> IDE) or 4 (in CI: `gradle test`, `xvfb-run` -> `gradle runIdeForUiTests` -> IDE) processes in parallel to manage

Reply:
![image](https://user-images.githubusercontent.com/3383210/88428865-f3409900-cdf5-11ea-8550-d053aabe5093.png)

Reply:
That's what I liked more about ide-probe (our in-house IDE testing framework): the test suite itself spawned the IDE process